File: foundation/foundation 3 month/lesson7/homework/ex_4.py
def check(ms):
	j=set(ms)
	k=sorted(j)
	return k	
k=int(input("K ni kiriting: "))
arr=list()
for x in range(k):
	s=input("Kiriting: ")
	arr.append(s)
print(arr)
print(check(arr))

File: foundation/foundation exams/exam foundation 2 month/ex_4.c
#include <stdlib.h>
#include <stdio.h>
void check(int k,int ms[k][k])
{	puts("");
	for(int i=0;i<k;i++)
	{
		for(int j=0;j<k;j++)
		{
			if(i<j)
			ms[i][j]=0;
			printf("%4d",ms[i][j]);
			
		}
	puts("");
	}
}
int main()
{
	int n;
	printf("n ni kiriting: ");
	scanf("%d",&n);
	int arr[n][n];
	for(int i=0;i<n;i++)
	{
		for(int j=0;j<n;j++)
		{
			scanf("%d",&arr[i][j]);
		}
		puts("");
	}
	check(n,arr);

}
